servicetalk-buffer-netty/src/test/java/io/servicetalk/buffer/netty/BufferAllocatorsTest.java

```diff
@@ -76,6 +84,42 @@ void testReadOnlyDirectBuffer(BufferAllocator allocator) {
         assertBuffer(allocator.wrap(ByteBuffer.allocateDirect(16).asReadOnlyBuffer()), true);
     }
 
```

Comment:
consider also testing negative case - i.e length is negative and/or offset is more than bytes length.

Reply:
Added `testWrapPartialByteArrayWithIllegalArguments`